pybleau/app/model/dataframe_plot_manager.py

```diff
@@ -62,6 +62,11 @@ class DataFramePlotManager(DataElement):
     each plot a PlotDescriptor is created to summarize the plot, and store it
     in contained_plots.
 
+    For consumers of this class, custom plot templates can be created and
+    accessed through an `IPlotTemplateInteractor`. A `DataFramPlotManager`
```

Comment:
How about something like this: "To enable creating plot templates and creating plots from them, a `template_interactor` attribute must be passed at creation. It must provide the `IPlotTemplateInteractor` interface, implementing how to save, load and delete Configurator objects/files."?

Reply:
Ok.